sdk/src/client/node_api/core/routes.rs

```diff
@@ -374,6 +374,21 @@ impl ClientInner {
             .await
     }
 
+    /// Get all UTXO changes of a given slot by slot commitment ID.
+    /// GET /api/core/v3/commitments/{commitmentId}/utxo-changes
+    pub async fn get_utxo_changes_by_slot_commitment_id(
+        &self,
+        slot_commitment_id: &SlotCommitmentId,
```

Comment:
why taking a `&`  to a type that is `Copy`?

Reply:
Still, 40 bytes is not 4. Just because it's Copy doesn't mean you can't use refs anymore.

Reply:
I know ... just wanted to make sure it's really due to performance.